content/00.front-matter.md

```diff
@@ -39,6 +39,10 @@ on {{manubot.date}}.
   {%- if author.twitter is defined and author.twitter is not none %}
     · ![Twitter icon](images/twitter.svg){.inline_icon}
     [{{author.twitter}}](https://twitter.com/{{author.twitter}})
+  {%- endif %}
+  {%- if author.mastodon is defined and author.mastodon is not none and author["mastodon-server"] is defined and author["mastodon-server"] is not none %}
+  · ![Mastodon icon](images/mastodon.svg){.inline_icon}
+  [\@{{author.mastodon}}@{{author.mastodon-server}}](https://{{author["mastodon-server"]}}/@{{author.mastodon}})
```

Comment:
```suggestion
  [\@{{author.mastodon}}@{{author["mastodon-server"]}}](https://{{author["mastodon-server"]}}/@{{author.mastodon}})
```
I think the build failed again because we missed this one